test/test-app/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "test-app",
+    "version": "1.0.0",
```

Comment:
> Before merging https://github.com/Azure/azure-functions-durable-js/pull/354 into this branch, the build pipeline failed (as expected) with this error (see [here](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=66438&view=logs&j=d25dc844-164d-552e-f1e9-8647caa62262&t=578a2197-efdf-58e0-879d-404438bf2588)):

Can you verify you did this for your final changes as well? It appears the link is for the first set of changes and just wanna make sure this still causes a build failure when expected

Reply:
Did this, see the `hossamnasr/build-pipeline-test-2` branch: https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=71235&view=results.